### ISSUE TITLE
Metamodeling literature

### ISSUE THREAD

**Issue by tanjaem** [slebok/slebok#17]
We should collect metamodeling literature for the reader

**Comment by grammarware** [comment]
Definitely! Everyone, please comment with pointers to papers and books we should not overlook.
